test/strategy_test.rb

```diff
@@ -452,7 +452,7 @@ def setup
     end
 
     test 'empty param' do
-      assert_equal nil, strategy.send(:signed_request_from_cookie)
+      assert_nil nil, strategy.send(:signed_request_from_cookie)
```

Comment:
Hey, mate. Should this be `assert_nil strategy.send(:signed_request_from_cookie)`?